Recipes Project.py

```diff
@@ -1,3 +1,18 @@
+#Recipe
+#marayah
+
+def recipes():
+    """
+    Description of the function
```

Comment:
Looks good! Perhaps you could start off with a small introduction of what the function is for. That might also make the input and output slightly easier to interpret.